test/helpers/execution.js

```diff
@@ -1,22 +1,21 @@
 const cql = require('cql-execution');
 const { PatientSource } = require('cql-exec-fhir');
 
-const MAIN_LIB_ID = 'mCODE';
-
 /**
  *
  * @param {Array} elmJSONs array of ELM JSON objects
  * @param {Object} patientBundle patient record to execute against
  * @param {Object} valueSetMap valueSetMap for CodeService
+ * @param {Object} libraryID the library ID of the cql library corresponding to the ELM
```

Comment:
```suggestion
 * @param {String} libraryID the library ID of the cql library corresponding to the ELM
```

Reply:
Commit `3eaedb6` should take care of both of your comments!